inc/options/class-wpseo-option-wpseo.php

```diff
@@ -51,7 +51,7 @@ class WPSEO_Option_Wpseo extends WPSEO_Option {
 		'environment_type'                         => '',
 		'content_analysis_active'                  => true,
 		'keyword_analysis_active'                  => true,
-		'inclusive_language_analysis_active'       => false,
+		'inclusive_language_analysis_active'       => true,
```

Comment:
This feature should not be enabled by default

Reply:
Fixed